index.js

```diff
@@ -431,6 +440,11 @@ exports.should_we_deny = function (next, connection, hook) {
 
 exports.hook_deny = function (next, connection, params) {
   const plugin = this;
+
+  if (plugin.should_we_skip(connection)) {
+    return next();
+  }
```

Comment:
Go ahead and use the one-line syntax on these:`if (plugin.should_we_skip(connection)) return true;`. The point of `should_we_deny()` is to encapsulate a [slightly] complex choice. If we ever opt to further complicate the choice, we'll do it in `should_we_deny()` rather than in the callers.